src/Lykke.Service.HFT/PeriodicalHandlers/GhostOrdersRemover.cs

```diff
@@ -28,8 +29,13 @@ public class GhostOrdersRemover : TimerPeriod
 
         public override async Task Execute()
         {
-            var ordersInOrderBookState = _orderStateRepository.All()
-                .Where(x => x.Status == OrderStatus.InOrderBook || x.Status == OrderStatus.Processing).ToList();
+            var chunkSize = 5000;
```

Comment:
chunkSize is initialised by 5000 two times in two different places in the code. Maybe worth adding configurable or constant value?